### DISCUSSION TITLE
Question about rust analyzer

### DISCUSSION THREAD

**Discussion by qvc0** [helix-editor/helix#9193] [Q&A] [answered]
What should i do if i installed rust analyzer manually through a command on rust website, but helix doesn't launch it when i edit rust files, it recommends me to download rust-analyzer with arch repository?

**Comment by kirawi** [comment, marked as answer]
`rust-analyzer` needs to be on `PATH`, and I don't think `rustup` adds it to `PATH`. You can either add it to `PATH` yourself or create a local `languages.toml` with

```toml
[language-server.rust-analyzer]
command = "rustup"
args = ["rust-analyzer"]
